src/foam/support/modal/NewEmailSupportModal.js

```diff
@@ -101,12 +105,19 @@ foam.CLASS({
       {
         class: 'Long',
         name: 'id'
+      },
+      {
+        class: 'String',
+        name: 'emailRegex',
+        value: /^(([^<>()[\]\\.,;:\s@\"]+(\.[^<>()[\]\\.,;:\s@\"]+)*)|(\".+\"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/
       }
     ],
 
     messages:[
       { name:'title', message:'New Email' },
       { name:'titlelabel', message:'Input the email address you want to connect to the help desk.' },
+      { name:'emailInvalid', message: 'The email you have entered is invalid, try again.'},
+      { name:'emailNotExist', message: 'The email you have entered is existed.'}
```

Comment:
There's a minor typo in the message here. Replace with the following: 'The email you have entered already exists.' Also the name of this message is a little confusing. Rename it to something more suggestive of the message like: "EmailExists".

Reply:
Sure